### PR TITLE
Add GitHub Packages installation instruction to the README

### PR DESCRIPTION
**Pull request by KacperKluka**
I've added the steps required to add AAT as a dependency in Gradle. It isn't as straightforward as one might think so it's great that we will have this information in our README :wink:

### CODE REVIEW

**Review by AndyNicks — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -55,12 +55,58 @@ GOOGLE_MAPS_API_KEY=create_api_key_with_geolocation_maps_sdk
 
 ### Maven / Gradle Dependencies
 
-We publish to [GitHub Packages](https://github.com/ably/ably-asset-tracking-android/packages/) for this repository.
-
 Kotlin users will want to add either `publishing-sdk` or `subscribing-sdk`, according to the needs of their project.
 Java users should add either `publishing-sdk-java` or `subscribing-sdk-java`.
 See [Android Runtime Requirements](#android-runtime-requirements) for more details.
 
+#### GitHub Packages
```
Comment (QuintinWillison) [inline README.md:62]: Vertical white space, please - we need a blank line below this heading.

File: README.md [hunk 2]
```diff
@@ -55,12 +55,58 @@ GOOGLE_MAPS_API_KEY=create_api_key_with_geolocation_maps_sdk
...
+We publish to [GitHub Packages](https://github.com/ably/ably-asset-tracking-android/packages/) for this repository.
+
+To install the dependency you will first need to create a "Personal Access Token" in your GitHub account. The token must have the `read:packages` permission.
```
Comment (QuintinWillison) [inline README.md:65]:
```suggestion
To install the dependency you will first need to create a "Personal Access Token" in your GitHub account. That token must have the `read:packages` permission.
```

File: README.md [hunk 3]
```diff
@@ -55,12 +55,58 @@ GOOGLE_MAPS_API_KEY=create_api_key_with_geolocation_maps_sdk
...
+Then use your generated token to authenticate in the AAT GitHub Packages repository:
```
Comment (QuintinWillison) [inline README.md:66]:
```suggestion
Then use your generated token to authenticate with the AAT GitHub Packages repository:
```

File: README.md [hunk 4]
```diff
@@ -55,12 +55,58 @@ GOOGLE_MAPS_API_KEY=create_api_key_with_geolocation_maps_sdk
...
+        credentials {
+            username = '<MAPBOX_USERNAME>'
+            password = '<MAPBOX_DOWNLOADS_TOKEN>'
+        }
+    }
+}
+```
+
+Finally, you can add the AAT dependency in the Gradle build script
+```gradle
```
Comment (QuintinWillison) [inline README.md:100]: Vertical white space - please can you insert a blank line above this code fenced block.

**Review by QuintinWillison — CHANGES_REQUESTED** [review]
I've spotted a few small things that could be improved.

Also, in terms of your reference to use of a PAT - is this really the only way to do this? What about those who may need to get this working from CI/CD environment - perhaps using the `GITHUB_TOKEN` issued for use by a workflow? Even if you don't document _how_ to do that specifically, I think you could enhance this documentation with:

1. An indication that PAT is just one way of achieving this - perhaps the easiest for personal development
2. At least one link back to GitHub's documentation where the use of GitHub packages is covered in respect of auth

File: README.md (hunk 4 above)
Comment (KacperKluka) [reply README.md:103]: https://github.com/ably/ably-asset-tracking-android/pull/523/commits/8128cf2d374e40d3a50f3f968dba4df0f6d7ae53

File: README.md (hunk 1 above)
Comment (KacperKluka) [reply README.md:62]: https://github.com/ably/ably-asset-tracking-android/pull/523/commits/8128cf2d374e40d3a50f3f968dba4df0f6d7ae53

File: README.md (hunk 3 above)
Comment (KacperKluka) [reply README.md]: https://github.com/ably/ably-asset-tracking-android/pull/523/commits/a45961cd3ba1b37410c8daec8809e47d8a4981ab

File: README.md (hunk 2 above)
Comment (KacperKluka) [reply README.md]: https://github.com/ably/ably-asset-tracking-android/pull/523/commits/a45961cd3ba1b37410c8daec8809e47d8a4981ab

**Comment by KacperKluka** [comment]
@QuintinWillison thanks for the suggestion :bow: I've added the information that you can use both `GITHUB_TOKEN` and PAT to authenticate https://github.com/ably/ably-asset-tracking-android/pull/523/commits/635982ec33b6e36957e3a14f0c7bcb4f51e60f0c :wink:

**Review by QuintinWillison — APPROVED** [review]

**Pull request merged**